src/presenters/admin/sidebar-presenter.php

```diff
@@ -56,7 +56,7 @@ class="attachment-full size-full content-visible"
 							<?php
 							if ( \YoastSEO()->classes->get( Promotion_Manager::class )->is( 'black-friday-2024-promotion' ) ) {
 								/* translators: %1$s and %2$s expand to a span wrap to avoid linebreaks. */
```

Comment:
/* translators: %1$s and %3$s expand to a span wrap to avoid linebreaks. %2$s expands to "Yoast SEO Premium". */